env-dist

```diff
@@ -21,7 +21,7 @@ OIDC_RP_CLIENT_SECRET_ENCODED = True
 OIDC_OP_DOMAIN = ''
 OIDC_OP_STORE_ACCESS_TOKEN = True
 # Assuming user remo with password remo and db name db_1
-DATABASE_URL = 'mysql://remo:remo@127.0.0.1:3306/db_1'
+DATABASE_URL = 'mysql://remo:remo@db:3306/remo'
```

Comment:
True but if you follow the instruction doesn't specify this change to let them work on development.
To find this fix I lost two days, so in case it is better to add a comment but all the volunteers will do this change to have the docker system working